src/openrct2-ui/WindowManager.cpp

```diff
@@ -294,7 +294,7 @@ class WindowManager final : public IWindowManager
             }
             case WindowClass::ScenarioSelect:
                 return ScenarioselectOpen(
-                    reinterpret_cast<scenarioselect_callback>(intent->GetPointerExtra(INTENT_EXTRA_CALLBACK)));
+                    reinterpret_cast<scenarioselect_callback>(intent->GetCloseCallbackExtra(INTENT_EXTRA_CALLBACK)));
```

Comment:
This type is only used in a few places. Would you mind renaming it `ScenarioSelectCallback` to follow current code style conventions?